keys/variables.tf

```diff
@@ -21,6 +16,11 @@ variable "concourse_workers_iam_role_arns" {
   description = "List of ARNs for the IAM roles that will be able to assume the role to access concourse keys in S3. Normally you'll include the Concourse worker IAM role here"
 }
 
+variable "generate_keys" {
```

Comment:
You don't seem to be using this variable?

Reply:
😱 you're right. Good catch, I'll fix it

Reply:
fixed